krane.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.4.0'
   spec.add_dependency("activesupport", ">= 5.0")
   spec.add_dependency("kubeclient", "~> 4.3")
-  spec.add_dependency("googleauth", "~> 0.8.0")
+  spec.add_dependency("googleauth", ">= 0.8.0")
```

Comment:
Can we do something like `['>= 0.8.0', "< 1"]`? I'd prefer not to risk someone installing a version with a breaking change.

Reply:
I missed this somehow. Sure! Updated the branch